### PR TITLE
2021-06-14 GUARD-1895 Square: Deduct Quantity For POS Sales

### PR DESCRIPTION
**Pull request by maxim-saltanov-skuvault**
Added LocationId and SourceName (POS or Online) fields - this can be determined by the presence / absence of the customer_id in the order object. Please see the [link](https://developer.squareup.com/forums/t/customer-id-field-locations-from-an-order/2030/2)

### CODE REVIEW

**Review by maxim-skuvault — COMMENTED** [review]
@maxim-saltanov-skuvault , Looks good besides the minor inline comment

File: src/SquareAccess/Models/Order.cs [hunk 1]
```diff
@@ -57,4 +61,10 @@ public static class SquareOrderState
 		public const string Open = "OPEN";
 		public const string Cancelled = "CANCELED";
 	}
+	
+	public enum SquareSourceNameEnum
+	{				
+		web,
```
Comment (maxim-skuvault) [inline src/SquareAccess/Models/Order.cs:67]: @maxim-saltanov-skuvault , Our convention is to capitalize enum values (Web, Pos), unless they have to be lower-case

Comment (maxim-saltanov-skuvault) [reply src/SquareAccess/Models/Order.cs]: fixed

**Review by maxim-skuvault — APPROVED** [review]

**Review by nathan-skuvault — APPROVED** [review]
Looks good!

**Review by jnewman-sv — APPROVED** [review]

**Pull request merged**